Update firewall rules, PrivateLink and VPC Peering documentation (#207)
cloudamqp/terraform-provider-cloudamqp: docs/resources/vpc_peering.md

This resouce allows you to accepting VPC peering request from an AWS requester. This is only available for CloudAMQP instance hosted in AWS.

> ~> **Note:** Creating a VPC peering will trigger a firewall change that automatically add rule for the peered subnet.

Reviewer: ```suggestion
~> **Note:** Creating a VPC peering will automatically add firewall rules for the peered subnet.
```

## Import

Not possible to import this resource.

## Create VPC Peering with additional firewall rules

To create a VPC peering configuration with additional firewall rules, it's required to chain the [cloudamqp_security_firewall](https://registry.terraform.io/providers/cloudamqp/cloudamqp/latest/docs/resources/security_firewall)

> resource to avoid parallel conflicting resource calls. This is done by adding dependency in the firewall resource to the VPC peering resource (`cloudamqp_vpc_peering.vpc_accept_peering`).

Reviewer: ```suggestion
resource to avoid parallel conflicting resource calls. You can do this by making the firewall resource depend on the VPC peering resource (`cloudamqp_vpc_peering.vpc_accept_peering`).
```
